src/payments/services/payment.service.ts

```
@@ -46,6 +50,89 @@ export class PaymentService {
+
+    const tenantId = this.configService.get<string>('DEFAULT_TENANT_ID');
+    const organisationId = this.configService.get<string>(
+      'DEFAULT_ORGANISATION_ID',
+    );
+    const lmsUrl = this.configService.get<string>('LMS_SERVICE_URL');
+    const academicYearId =
+      this.configService.get<string>('DEFAULT_ACADEMIC_YEAR_ID') || undefined;
+
+    if (!lmsUrl || !tenantId || !organisationId) {
+      throw new BadRequestException(
+        'Course pricing cannot be verified: configure LMS_SERVICE_URL, DEFAULT_TENANT_ID, and DEFAULT_ORGANISATION_ID.',
```

ESLint

[error] Replace `'Course·pricing·cannot·be·verified:·configure·LMS_SERVICE_URL,·DEFAULT_TENANT_ID,·and·DEFAULT_ORGANISATION_ID.',` with `"Course·pricing·cannot·be·verified:·configure·LMS_SERVICE_URL,·DEFAULT_TENANT_ID,·and·DEFAULT_ORGANISATION_ID."`

(prettier/prettier)